### ISSUE TITLE
Convert multi area cortical model to standardised format

### ISSUE THREAD

**Issue by pgleeson** [NeuralEnsemble/Networks_SIG#3]
The multi area model of macaque visual cortex ([Schmidt et al., 2018](https://link.springer.com/article/10.1007/s00429-017-1554-4)) is currently implemented in [NEST format](https://github.com/INM-6/multi-area-model), and would be an excellent option for conversion to standardised format, e.g. in NeuroMLlite. 

An updated version of the Potjans and Diesmann (2014) model would be a first step towards this: https://github.com/NeuroML/NeuroMLlite/blob/master/examples/Example6.py

**Comment by albada** [comment]
Good idea. See also the recent [PLOS CB publication](https://journals.plos.org/ploscompbiol/article/metrics?id=10.1371/journal.pcbi.1006359) about the dynamics of the model.
